client/messaging/common-message-layout.test.tsx

```diff
@@ -0,0 +1,328 @@
+import React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+import { SbUserId } from '../../common/users/user-info'
+import { TextMessage } from './common-message-layout'
+
+describe('client/messaging/common-message-layout/TextMessage', () => {
+  const renderer = ShallowRenderer.createRenderer()
+  const doMatch = (text: string): React.ReactElement => {
+    renderer.render(
+      <TextMessage userId={2 as SbUserId} selfUserId={1 as SbUserId} time={0} text={text} />,
```

Comment:
3 things:
1) don't cast to `SbUserId` like this, use `makeSbUserId`, that's what it's for
2) The value for `selfUserId` here should probably be pulled into a constant at the top level so it can be easily referenced in tests that want to check highlighting behavior
3) You don't have tests for highlighting behavior atm (in the case that the message *is* highlighted, at least), and those would definitely be good to have